CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### App Center
 
+* **[Feauture]** App Center SDK now supports Apple Silicon.
```

Comment:
Please add a note that Simulators for arm64 is NOT available via CocoaPods. They should use other methods intead (if needed)